src/app/infrastructure/http/auth.interceptor.ts

```diff
@@ -0,0 +1,24 @@
+import {
+  HttpRequest,
+  HttpInterceptor,
+  HttpHandler,
+  HttpEvent,
+} from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+
+const API_KEY = '09f328da5aa5d91f51978506a62cc80e';
+
+@Injectable()
+export class AuthInterceptor implements HttpInterceptor {
+  intercept(
+    req: HttpRequest<any>,
```

Comment:
Hi George!
What do you think about `any` as types for HttpRequest and HttpEvent body?